package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "mongo-models",
-  "version": "2.0.3",
+  "version": "2.1.0",
   "description": "JavaScript class interfaces to MongoDB collections",
   "main": "index.js",
   "scripts": {
     "example": "MONGODB_URI='mongodb://localhost:27017' MONGODB_NAME='mongo-models-test' PORT=8080 node example/server.js",
-    "test": "lab -c -L --assert code",
+    "test": "lab -c -L -m 5000 --assert @hapi/code",
```

Comment:
Why 5 seconds? Is something taking longer than the 2 second default?

Reply:
Yes, test 2 'it throws when the db connection fails' was timing out before it could complete. It's taking 3 to 4 seconds for the test to complete on my system.

Reply:
Let's try reverting and see if it fails the build because of the timeout.

Reply:
How about just a longer timeout on test that times out?